omero/sysadmins/server-upgrade.txt

```diff
@@ -104,7 +104,17 @@ so that the app is available from the context menu on the webclient tree::
     $ bin/omero config append omero.web.open_with '["omero_figure", "new_figure",
       {"supported_objects":["images"], "target": "_blank", "label": "OMERO.figure"}]'
 
+Updating 'Open with' config
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+If you have configured :property:`omero.web.open_with` prior to OMERO 5.3.3 and
+also set the default viewer with :property:`omero.web.viewer.view`, for example
+as described for `OMERO.iviewer <https://pypi.python.org/pypi/omero-iviewer>`_
+then you will find that ``Open with > Image Viewer`` also opens the OMERO.iviewer
+instead of the ``webgateway`` viewer.
+
+To fix this, you need to manually update ``open_with`` as
+`described on this PR <https://github.com/openmicroscopy/openmicroscopy/pull/5317/>`_.
```

Comment:
I'd strongly suggest that the example be put here in this page and not require users to go to the PR. We should be the ones to go the extra step, not the users. For comparison, I don't see any other mention of a PR in this repo:

```
$ git grep -E "github.*pull" -- .
common/_ext/edit_on_github.py:Loosely based on https://github.com/astropy/astropy/pull/347
contributing/using-git.txt:(PRs) <https://github.com/openmicroscopy/openmicroscopy/pulls>`_. These
contributing/using-git.txt:-  `Using Pull Requests <http://help.github.com/articles/using-pull-requests>`_
contributing/using-git.txt:-  `Pull Requests 2.0 <https://github.com/blog/712-pull-requests-2-0>`_
omero/index.txt:.. _Pull Requests: http://help.github.com/articles/using-pull-requests
```